libs/coin-modules/coin-solana/src/api/chain/index.ts

```diff
@@ -129,7 +129,24 @@ const remapErrorsWithRetry = <P extends Promise<T>, T>(callback: () => P, times
   };
 };
 
-const kyNoTimeout = ky.create({ timeout: false });
+/*
+NOTE: https://github.com/sindresorhus/ky?tab=readme-ov-file#retry
+defaults values are set here https://github.com/sindresorhus/ky/blob/b49cd03d8673ea522a29bae4ef6b4672cf23201b/source/utils/normalize.ts#L14
+retrying on 404 is debatable, but due to current solana node behavior, it's necessary
+*/
+const kyNoTimeout = ky.create({
+  timeout: false,
+  retry: {
+    limit: 3,
+    statusCodes: [404, 408, 413, 429, 500, 502, 503, 504],
+    methods: ["get", "post"],
```

Comment:
We should keep the rest of the defaults as options request can also fail sometimes
```suggestion
    methods: ["get", "post", "put", "head", "delete", "options", "trace"],
```

Reply:
done ✅